index.html

```diff
@@ -9064,8 +9050,454 @@
+	function wrapperSpan(role, type) { let textStyle = as[`${type}_color_${role}`]; return `<span style='color: ${textStyle.color}; font-style: ${textStyle.fontStyle}; font-weight: ${textStyle.fontWeight}'>$1</span>`; }
+
+	function applyStylizedCodeBlocks() {
+		let blocks = newbodystr.split(/(```.*?\n[\s\S]*?```)/gs);
+		for (var i = 0; i < blocks.length; i++) {
+			if (blocks[i].startsWith("```")) { blocks[i] = blocks[i].replaceAll(/```.*?\n([\s\S]*?)```/gs, `</p><pre style='min-width:90%;margin:0px 100px 0px 20px;background-color:${as.code_block_background};color:${as.code_block_foreground}'>$1</pre><p>`); }
+			else { blocks[i] = blocks[i].replaceAll("```","`").replaceAll("``","`").replaceAll(/`(.*?)`/g, `<code style='background-color:black'>$1</code>`);}
+		}
+		return blocks.join('');
+	}
+}
+
+function updateTextPreview() {
+	let preview = `You are Mikago, a prestigious bot that's a supervillain.\n\nRoleplay in first person, be prestigious, don't be a bot. This is a fantasy world.\n\nCode blocks should be wrapped in triple backticks, like so:\nqqq\n<Some_\n-- multiline\n--- code here$\nqqq\n[AI_REPLY]\n*takes my hat off to greet the squad* "Greetings, I am Mikago, the prestigious!" *bows to the crew*\n*clears my throat* "Now, I'm sure there are many questions, but all will be answered in due time." *deep breath*\n[USER_REPLY]\n*raises eyebrow and looks at you* "Hm.. you don't seem very prestigious.. How do we know you're Mikago?"\nI check with the crew and we decide to test Mikago with a piece of info they should know.\n"Ok.. now, if you're REALLY Mikago, you would know the answer to this.." *pauses for dramatic effect* "How do you type code?"\n[AI_REPLY]\n*looks at you* "You serious?"\n[USER_REPLY]\n*draws my sword* "Yes. You should know the code to calculate the factorial of a number."\nThe crew also draws their weapons and point them at you, not giving you any space.\n[AI_REPLY]\n*backs off* "Woah, easy there.." *makes some steps backwards, but then stops*\n"I would normally take this as an insult to my prestige, but I understand your caution.." *takes a deep breath*\n"Well, if it's to prove myself, here goes the python code to calculate the factorial of a number.."\n\nMikago opens a live-code-portal with his magic and writes the code that was requested.\nqqq\ndef factorial(n):\n    if n == 0: return 1\n    else: return n * factorial(n-1)\nqqq\n*looks at you, getting impatient* "Are we ok now.. or do you want me to write the code of a game next?"\n[USER_REPLY]\n*sheathes my sword and approaches for a hug* "Oh, Mikago, my old friend, it is really you!"`;
+	document.getElementById('aesthetic_text_preview').innerHTML = render_enhanced_chat_instruct(preview.replaceAll('qqq', '```'));
+}
+</script>
+
```

Comment:
The customization panel is having issues with Responsive layouts - it does not adjust to fit different aspect ratios and is unusable on a mobile resolution. See attached:

![image](https://github.com/LostRuins/lite.koboldai.net/assets/39025047/824ec428-474e-4e72-b2ca-1ebb101104ed)